src/wireviz/wv_bom.py

```diff
@@ -156,26 +157,28 @@ def component_table_entry(
         pn: Optional[str] = None,
         manufacturer: Optional[str] = None,
         mpn: Optional[str] = None,
+        supplier: Optional[str] = None,
+        spn: Optional[str] = None,
     ) -> str:
     """Return a diagram node table row string with an additional component."""
-    manufacturer_str = manufacturer_info_field(manufacturer, mpn)
+    manufacturer_str = pn_info_string("mpn", manufacturer, mpn)
+    supplier_str = pn_info_string("spn", supplier, spn)
+    part_number_list = [pn, manufacturer_str, supplier_str]
```

Comment:
Consider alternative:
```suggestion
    part_number_list = [pn, pn_info_string(HDR_MPN, manufacturer, mpn), pn_info_string(HDR_SPN, supplier, spn)]
```
or maybe:
```suggestion
    part_number_list = [
        pn,
        pn_info_string(HDR_MPN, manufacturer, mpn),
        pn_info_string(HDR_SPN, supplier, spn),
    ]
```
unless readability is hurt.